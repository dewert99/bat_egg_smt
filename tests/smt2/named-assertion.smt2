(declare-const a Bool)
(declare-const b Bool)
(declare-const c Bool)
(assert (! a :named an))
(assert (! b :named bn))
(assert (! c :named cn))
(check-sat)
(get-model)
(check-sat-assuming ((not (and an bn))))
(get-unsat-core)