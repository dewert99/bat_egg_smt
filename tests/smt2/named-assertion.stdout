sat
(
 (define-fun a () Bool true)
 (define-fun an () Bool true)
 (define-fun b () Bool true)
 (define-fun bn () Bool true)
 (define-fun c () Bool true)
 (define-fun cn () Bool true)
)
unsat
((not (and an bn)) bn an)
